src/KubeLibrary/KubeLibrary.py

```diff
@@ -611,6 +611,27 @@ def get_secrets_in_namespace(self, name_pattern, namespace, label_selector=""):
         secrets = [item for item in ret.items if r.match(item.metadata.name)]
         return secrets
 
+    def get_namespaced_pod_exec(self, name, namespace, cmd):
+        """Exec command on selected container for POD.
+
+        Returns command stdout/stderr
+
+        - ``name``:
+          pod name
+        - ``namespace``:
+          namespace to check
+        - ``command``:
+          command to be executed
+        """
+        return stream.stream(self.v1.connect_get_namespaced_pod_exec,
+                             name,
+                             namespace,
+                             command=["/bin/sh", "-c", cmd],
```

Comment:
I think we shouldn't hardcode sh here, there should be freedom of using whatever you want + support for multiple commands (using &&, ;, etc.)

Reply:
I have added an option to pass "argv" syntax value in cmd, so we can easily pass it with && or ;
I guess that for now, that's enough, we can work on improvements later then :)